### PR TITLE
Add "invincible" as a synonym of "invulnerable" in related syntaxes.

### PR DESCRIPTION
**Pull request by sovdeeth**
### Description

Syntaxes that use "invulnerable" will also allow "invincible" as well, to reduce confusion and make it slightly more likely someone can write the syntax from memory.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/effects/EffInvulnerability.java [hunk 1]
```diff
@@ -40,8 +40,8 @@ public class EffInvulnerability extends Effect {
 	
 	static {
 		Skript.registerEffect(EffInvulnerability.class,
-			"make %entities% invulnerable",
-			"make %entities% (not invulnerable|vulnerable)");
+			"make %entities% (invulnerable|invincible)",
+			"make %entities% (not (invulnerable|invincible)|vulnerable)");
```
Comment (AyhamAl-Ali) [inline src/main/java/ch/njol/skript/effects/EffInvulnerability.java:44]: I think `vincible` works here too right?
```suggestion
			"make %entities% (not (invulnerable|invincible)|vulnerable|vincible)");
```

**Review by AyhamAl-Ali — APPROVED** [review]

File: src/main/java/ch/njol/skript/effects/EffInvulnerability.java (hunk 1 above)
Comment (sovdeeth) [reply src/main/java/ch/njol/skript/effects/EffInvulnerability.java:44]: is that a real word?

Comment (AyhamAl-Ali) [reply src/main/java/ch/njol/skript/effects/EffInvulnerability.java:44]: I searched it up on Google, look like it 😬

Comment (sovdeeth) [reply src/main/java/ch/njol/skript/effects/EffInvulnerability.java:44]: i mean sure i guess i could add it
I doubt anyone will ever use it though haha

**Review by APickledWalrus — APPROVED** [review]

**Comment by Pikachu920** [comment]
i like the change. can we also add invincible as doc keywords (`@Keywords`)

**Comment by AyhamAl-Ali** [comment]
> i like the change. can we also add invincible as doc keywords (`@Keywords`)

no need to since it is in the pattern now and the docs search does look in patterns for matches :D

File: src/main/java/ch/njol/skript/effects/EffInvulnerability.java [hunk 2]
```diff
@@ -40,8 +40,8 @@ public class EffInvulnerability extends Effect {
 	
 	static {
 		Skript.registerEffect(EffInvulnerability.class,
-			"make %entities% invulnerable",
-			"make %entities% (not invulnerable|vulnerable)");
+			"make %entities% (invulnerable|invincible)",
+			"make %entities% (not (invulnerable|invincible)|vulnerable|vincible)");
```
Comment (Moderocky) [inline src/main/java/ch/njol/skript/effects/EffInvulnerability.java:44]: At this point couldn't we just have a flag for [in]?

**Review by Moderocky — APPROVED** [review]

File: src/main/java/ch/njol/skript/effects/EffInvulnerability.java (hunk 2 above)
Comment (sovdeeth) [reply src/main/java/ch/njol/skript/effects/EffInvulnerability.java:44]: Sure, but I left it like this because I think it's easier to read for users.

Comment (AyhamAl-Ali) [reply src/main/java/ch/njol/skript/effects/EffInvulnerability.java:44]: Also it's helpful for docs searching instead of using `@Keywords`

**Pull request merged**